jwql/utils/credentials.py

```diff
@@ -0,0 +1,49 @@
+"""Utility functions related to accessing remote services and databases.
+
+Authors
+-------
+
+    - Johannes Sahlmann
+
+Use
+---
+
+    This module can be imported as such:
+
+    >>> import credentials
+    token = get_mast_token()
+
+ """
+import os
+
+from astroquery.mast import Mast
+
+from jwql.utils.utils import get_config, check_config
+
+
+def get_mast_token(request=None):
+    """Return MAST token."""
```

Comment:
Please provide a `Parameters` section to the docstring that describes `request`

Reply:
Whoops, I completely forgot to add this docstring. Thanks for the reminder!